SignalServiceKit/src/Protocols/ProfileManagerProtocol.h

```diff
@@ -16,5 +16,6 @@
 - (BOOL)isThreadInProfileWhitelist:(TSThread *)thread;
 
 - (void)addUserToProfileWhitelist:(NSString *)recipientId;
+- (void)addGroupIdToProfileWhitelist:(NSData *)groupId;
```

Comment:
This is fine but we could also use `addThreadToProfileWhitelist:` if its more convenient/concise.